lib/src/clients.dart

```diff
@@ -9,6 +9,11 @@ import 'dart:convert';
 
 import 'package:http/http.dart' as http;
 
+import 'forbidden_headers_io.dart'
+    // ignore: uri_does_not_exist
+    if (dart.library.html) 'forbidden_headers_js.dart'
+    // ignore: uri_does_not_exist
+    if (dart.library.io) 'forbidden_headers_io.dart';
```

Comment:
I think conditional imports is a too big hammer for this.

You can get the necessary information by:
```
const forbiddenHeaders = bool.fromEnvironment('dart.library.html') ? <String>['user-agent', 'content-length'] : <String>[];
```